pages/en/glossary.mdx

```diff
@@ -0,0 +1,81 @@
+
+- **Subgraph consumers**: Any application or user that queries a subgraph.
+
+- **Subgraph developer**: A developer who builds and deploys a subgraph to The Graph's decentralized network.
+
+- **Subgraph manifest**: A JSON file that describes the subgraph's GraphQL schema, data sources, and other metadata.
+
+- **Rebate pool**: An economic security measure that holds query fees paid by subgraph consumers. Indexers must wait for an epoch to close before they can claim their query fee rebates.
+
+- **Epoch**: A period of time that is used to measure the performance of Indexers. At the end of an epoch, Indexers are rewarded for their performance.
+
+- **Allocation**: An Indexer can allocate self and delegated stake towards subgraphs that have been published on The Graph's decentralized network. This is called opening an allocation, and indicates to the network that the Indexer is actively indexing and serving queries to the subgraph. Open allocations accrue indexing rewards proportional to the signal on the subgraph, and the amount of GRT allocated. An Indexer may claim the accrued indexing rewards on a given subgraph by submitting a recent, and valid, Proof of Indexing (POI). This is known as closing an allocation. The maximum allocation period is 28 epochs. If an indexer leaves an allocation open beyond 28 epochs, it is known as a stale allocation. Allocations exist in one of four phases:
+
+  1. **Active**: An allocation is considered active when it is created on-chain. The Indexer agent manages creating allocations based on the Indexer rules.
+
+  2. **Closed**: An Indexer can close an allocation once an epoch has passed, or their Indexer agent will automatically close the allocation after ~26 days. When an allocation is closed, their indexing rewards are distributed to the Indexer and its Delegators.
+
+  3. **Finalized**: Once an allocation has been closed, there is a dispute period, allowing an opportunity for a fisherman to open a dispute.
+
+  4. **Claimed**: The final phase of an allocation, all eligible rewards have been distributed and query fee rebates have been claimed.
+
+- **Subgraph Studio**: A powerful dapp for building, deploying, and publishing subgraphs.
+
+- **Fishermen**: Network participants that monitor query results served by Indexers. Fishermen are rewarded with GRT and receive 50% of the Indexer's indexing rewards for that epoch if they find an Indexer serving false data.
+
+- **Arbitrators**: Arbitrators guarantee network security by resolving disputes between Indexers and Fishermen.
+
+- **Slashing**: Indexers can have their rewards slashed for serving false data. The Indexer's self stake is slashed by 2.5% and the entirety of the rewards for that epoch are slashed as well. Additionally, 50% of the Indexer’s indexing rewards for that epoch are burned, and the other 50% are rewarded to the Fisherman.
```

Comment:
> the entirety of the rewards for that epoch are slashed as well. Additionally, 50% of the Indexer’s indexing rewards for that epoch are burned, and the other 50% are rewarded to the Fisherman

Is it really "additionally"? Because it seems like we're talking about the same rewards: they are slashed, because 50% of it is burned and 50% goes to the fisherman. No?

Reply:
Agreed, doesn't make much sense. Removed it!